selftests/unit/test_utils_partition.py

```diff
@@ -45,12 +37,11 @@ class TestPartition(unittest.TestCase):
 
     @unittest.skipIf(missing_binary('mkfs.ext2'),
                      "mkfs.ext2 is required for these tests to run.")
-    @unittest.skipIf(missing_binary('sudo'),
-                     "sudo is required for these tests to run.")
-    @unittest.skipIf(cannot_sudo('mount'),
+    @unittest.skipIf(not process.can_sudo('mount'),
```

Comment:
The force umount test requires running kill as a privileged user (by the underlying libs):

```diff
diff --git a/selftests/unit/test_utils_partition.py b/selftests/unit/test_utils_partition.py
index 9aa0727..b698134 100644
--- a/selftests/unit/test_utils_partition.py
+++ b/selftests/unit/test_utils_partition.py
@@ -59,6 +59,8 @@ class TestPartition(unittest.TestCase):
         self.disk.unmount()
         self.assertNotIn(self.mountpoint, open("/proc/mounts").read())
 
+    @unittest.skipIf(not process.can_sudo('kill'),
+                     "requires running kill as a privileged user")
     def test_force_unmount(self):
         """ Test force-unmount feature """
         self.disk.mkfs()
```